docs/man-pages/condor_token_request_approve.rst

```diff
@@ -0,0 +1,110 @@
+
+
+*condor_token_request_approve*
+======================
+
+approve a token request at a remote daemon
+:index:`condor_token_request_approve<single: condor_token_request_approve; HTCondor commands>`\ :index:`condor_token_request_approve command`
+
+Synopsis
+--------
+
+**condor_token_request_approve** [**-reqid** *val*]
+[**-pool** *pool_name*] [**-name** hostname] [**-type** *type*]
+[**-debug**]
+
+**condor_token_request** [**-help** ]
+
+Description
+-----------
+
+*condor_token_request_approve* will approve an request for an authentication token
+queued at a remote daemon.  Once approved, the requester will be able to fetch a
+fully signed token from the daemon and use it to authenticate with the TOKEN method.
+
+**NOTE** that any user can request a very powerful token, even allowing them to be
```

Comment:
This sounds a little too alarming. The user cannot get _additional_ perms for the token than they would already have from the various ALLOW* knobs.

Reply:
I tweaked it a bit, but I think it's the right amount of alarming.  The user can request to be `condor@foo` which likely has all the necessary authorizations.

In my next update I explicitly mention users can request any identity instead of authorization.  Hopefully that avoids the above confusion.